pre_submit.csh

```diff
@@ -0,0 +1,208 @@
+#!/bin/tcsh -f
+
+# > > > Add code to check whether start_ end_ years are different
+#       and whether user_nl_docn_# have taxmode turned on or off?
+#       I want it turned on at the start, when the years are the same,
+#        and off at the end when they're different
+
+# Script to submit a variety of jobs:
+#    full cycle(s)
+#    assim only
+#    flexible numbers of cycles/job and resubmissions
+# and to check whether the case is ready for that submission:
+#    prevent running assimilations when there are too many cesm.log files in $rundir
+#    check the requested start date against the one in rpointer.atm_0001
+#    
+# Run in $CASEROOT
+
+if ($#argv == 0) then
+   echo "Usage: submit first_date last_date cycles_per_job queue"
+   echo "       first_date, last_date = YYYY-MM-DD-SSSSS"
+   echo "       first_date = last_date -> assimilation only."
+   echo "       cycles_per_job: The number of jobs will be calculated from the dates"
+   echo "                       Set to 1 for assimilation only jobs."
+   echo "       If 'queue' is omitted, the wall clock will be calculated, then exit"
+   exit
+endif
```

Comment:
This should be if all three arguments are not supplied.

Reply:
Actually there are 4.  I don't reference the fourth until line 169,
which is mockable code.  That line could be moved up to where
the other 3 args are used.
And the Usage if test can be changed to != 4.